projects/admin/src/app/record/detail-view/holding-detail-view/serial-holding-detail-view/serial-holding-detail-view.component.ts

```diff
@@ -185,5 +160,49 @@ export class SerialHoldingDetailViewComponent implements OnInit {
       }
     );
   }
+
+  // COMPONENT PRIVATE FUNCTIONS ==============================================
+  /**
+   * Check permissions to know if current user could manage issue creation.
+   *   To be able to create an issue, the current logged user must have
+   *   ITEM_CREATE permission and should be able to create an issue for this
+   *   holding. This last check could be determined checking the holding
+   *   `update` permissions.
+   */
+  private _computePermissions(): void {
+    if (this._permissionsService.canAccess(PERMISSIONS.ITEM_CREATE)) {
+      this._recordPermissionService
+        .getPermission('holdings', this.holding.metadata.pid)
+        .subscribe((data: RecordPermissions) => {
+          this.recordPermissions = data;
+          this.allowIssueCreation = data?.update?.can;
```

Comment:
Does `data?.update?.can` return false, if there is no value?

Reply:
you're right ! I will change my code.